src/include/migraphx/stringutils.hpp

```diff
@@ -177,6 +177,18 @@ inline std::string interpolate_string(const std::string& input,
         std::move(end));
 }
 
+inline std::string to_c_id(const std::string& name, char rep = '_')
```

Comment:
to_c_id = to_concat_id?

Reply:
No, it converts the string to a C identifier, hence `to_c_id`. It was moved from module.cpp which was used to print the IR. Its moved here so it can also be used for codegen to ensure that all the variables and template parameters we have are all valid C identifiers as well.